vmm/src/lib.rs

```diff
@@ -1225,8 +1225,11 @@ impl Vmm {
             error!("Failed to log metrics while stopping: {}", e);
         }
 
-        // Exit from Firecracker using the provided exit code.
-        std::process::exit(exit_code);
+        // Exit from Firecracker using the provided exit code. Safe because we're terminating
+        // the process anyway.
+        unsafe {
+            libc::_exit(exit_code);
```

Comment:
But why is it better to terminate the process with libc::exit? Feel free to add the explanation to the commit message as well.